python/setup.py

```diff
@@ -120,7 +120,7 @@ def build_extension(self, ext) -> None:
         build_ext.build_extension(self, ext)
 
 
-setup(name='intel-ittapi',
```

Comment:
Unchanged `intel-ittapi` project name still appears in Readme.md in several places

Reply:
changed